pkg/deploy/api/v1/types.go

```diff
@@ -278,6 +278,8 @@ const (
 // DeploymentTriggerImageChangeParams represents the parameters to the ImageChange trigger.
 type DeploymentTriggerImageChangeParams struct {
 	// Automatic means that the detection of a new tag value should result in a new deployment.
+	// Note that if this is set to false, the image field in the container specification will
+	// not be automatically resolved to an image stream tag specified in field from.
 	Automatic bool `json:"automatic,omitempty" description:"whether detection of a new tag value should trigger a deployment"`
```

Comment:
Add that comment to description as well, this way we'll have it in the swagger spec ppl are looking at.